migrations/versions/e9783faa5bb0_events_and_issues.py

```diff
@@ -0,0 +1,66 @@
+"""events and issues
+
+Revision ID: e9783faa5bb0
+Revises: 362b786c5916
+Create Date: 2022-03-03 13:48:42.580271
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'e9783faa5bb0'
+down_revision = '362b786c5916'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('events',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('name', sa.String(length=255), nullable=False),
+    sa.Column('short_description', sa.String(length=255), nullable=False),
+    sa.Column('long_description', sa.Text(), nullable=True),
+    sa.Column('is_active', sa.Boolean(), nullable=True),
+    sa.Column('is_complete', sa.Boolean(), nullable=True),
+    sa.Column('oh_attendance', sa.Integer(), nullable=True),
```

Comment:
outcome_id and oh_attendance fields are missing. Out of these, I have no idea about the oh_attendance field. Can you note this and ask Geoff about this.

Reply:
My understanding is `oh_attendance` is the number of people who attended Open House, applicable only if it is an open house event.